src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyIFormatProvider.cs

```diff
@@ -206,7 +206,7 @@ public override void Initialize(AnalysisContext analysisContext)
                         {
                             var semanticModel = argument.SemanticModel;
 
-                            var symbol = semanticModel.GetSymbolInfo(argument.Value.Syntax).Symbol;
+                            var symbol = semanticModel.GetSymbolInfo(argument.Value.Syntax, oaContext.CancellationToken).Symbol; // Does it matter here to use csaContext.CancellationToken or oaContext.CancellationToken?
```

Comment:
@mavasani, This is the only comment left now.

Reply:
Please always use innermost context's cancellation token. Analyzer driver is free to invoke each analyzer callback with a different cancellation token, and in fact it does happen in the IDE execution. I have seen unhandled operation cancelled exceptions from analyzers not using the correct cancellation token.